README.md

```diff
@@ -49,7 +49,7 @@ class Feature < ActiveRecord::Base
   include Flip::Declarable
 
   # The recommended Flip strategy stack.
-  strategy Flip::CookieStrategy
+  strategy Flip::CookieStrategy # alternatively can use strategy Flip::CookieStrategy for session-based
```

Comment:
There's a copy-paste error in this string.

Perhaps this is better:

`# Alternative: Use Flip::SessionStrategy to use Rails' encrypted sessions instead of cookies`